instrumentation/apolloconfig-apolloclient-1.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.ctrip.framework.apollo")
+    module.set("apollo-client")
+    versions.set("[1.0.0,2.3.0]")
```

Comment:
`2.3.0` is currently the latest release. Limiting the version here would only make sense if you know that your instrumentation is not going to work with the next version. I think you should be using `versions.set("[1.0.0,)")` here.

Reply:
The next version is not supported, and the specific release plan is not clear yet.

Reply:
I think you shouldn't really worry about it at this point. Our daily build will start failing when the breakage happens and at that point we'll either fix the instrumentation or limit the versions to get the build passing.

Reply:
OK，I'll change it